inc/trilogy/protocol.h

```diff
@@ -597,6 +604,128 @@ typedef struct {
     size_t data_len;
 } trilogy_value_t;
 
+typedef struct {
+    bool is_null;
+    bool is_unsigned;
```

Comment:
I think with moving this to the end of the struct and then also moving `is_negative` to the end for date / date time might allow for tighter packing of the struct?